src/common/store/gameFormats.js

```diff
@@ -41,6 +50,8 @@ export const NormalGameFormat = new (class extends GameFormat {
   displayName = 'Normal';
   description = 'Each player has a 30-card deck. No restrictions on cards.';
 
+  isDeckValid = deck => deckHasNCards(deck, DECK_SIZE);
```

Comment:
Nit: maybe rename to deckCorrectSize or something less ambiguous?

Reply:
I like being more explicit here because we could in principle have formats with different deck sizes (although that would make the `DECK_SIZE` constant misleading).